lib/gh/merge_commit.rb

```diff
@@ -58,9 +58,9 @@ def has_merge_commit?(hash)
 
     def github_done_checking?(hash)
       case hash['mergeable_state']
-      when 'checking'                               then false
-      when 'unknown'                                then hash['merged']
-      when 'clean', 'dirty', 'unstable', 'stable'   then true
+      when 'checking'                                                  then false
+      when 'unknown'                                                   then hash['merged']
+      when 'clean', 'dirty', 'unstable', 'stable', 'blocked', 'behind' then true
       else fail "unknown mergeable_state #{hash['mergeable_state'].inspect} for #{url(hash)}"
```

Comment:
This is an aside since this is existing code, but is there some way to “escalate” or somehow mark this kind of failure? Maybe people could have figured out the problem more quickly if this were urgently attention-getting.